docs/tour/dns.md

```diff
@@ -0,0 +1,216 @@
+# Setting up DNS with kops
+
+kops relies on configuring Route53 (on AWS) for internal discovery and so you can discover the API server
```

Comment:
Kops relies on configuring! Or does configure and uses dns for etcd and other needs